jenkins/upgrade_python_requirements.py

```diff
@@ -33,7 +34,17 @@
     help="The github organization for the repository to run make upgrade on.",
     required=True,
 )
-def main(sha, repo_root, org):
+@click.option(
+    '--user_reviewers',
+    help="Comma seperated list of Github users to be tagged on PR's",
```

Comment:
nit: I don't think the apostrophe is needed in "PRs", but not 100% positive on that.  Maybe just expand to "pull requests" for clarity?